src/main/java/Controllers/ChecklistManagerController.java

```diff
@@ -215,6 +215,20 @@ protected void addNewTask(ActionEvent actionEvent) {
         stage.close();
     }
 
+    /**
+     * Removes the selected Task from the currently selected Checklist
+     * @param actionEvent on click
+     */
+    @FXML
+    protected void deleteSelectedTask(ActionEvent actionEvent){
+        TaskManager taskManager = new TaskManager();
+        Task currTask = Data.getChecklistList().get(Data.getChecklistListIndex()).incomplete.get(Data.getChecklistListIndex());
```

Comment:
I'm not 100% sure about this line. Which task are we trying to delete here? This implementation deletes the task at at checklistListIndex, but don't we want to delete the task that is selected (highlighted) by the user in the manage checklist menu?